### DISCUSSION TITLE
Second day of the year, decline in month but increase in year score?

### DISCUSSION THREAD

**Discussion by kefiros** [iSoron/uhabits#1608] [Help & FAQ] [answered]
I am trying to understand habit statistics. Today is the 2nd of January. My habit score is showing a 11% monthly decline but a 8% yearly increase. How can you have a decline and increase all the same time, and after only 1 day? Is the reporting  for the CALENDAR month and year as the charts suggest our the LAST 30 and 365 days?

![Screenshot_20230102-101358](https://user-images.githubusercontent.com/7994428/210188018-8658deb7-a8a3-45a4-83c0-666a1c91dcb1.png)
![Screenshot_20230102-103316](https://user-images.githubusercontent.com/7994428/210188019-2fb8e7c3-03f1-4ae4-b2f1-5ff9baf1d96a.png)

**Comment by iSoron** [comment, marked as answer]
Compared to a month ago (30 days) your habit strength is 11 percentage points lower, but compared to one year ago (365 days), it is 8 percentage points higher. This can happen if the strength fluctuates up and down during the period, instead of increasing steadily. These charts shows rolling months/years.
